runOrthoinference.sh

```diff
@@ -9,11 +9,15 @@ git pull
 ## Create new jar file with orthoinference code
 mvn clean compile assembly:single
 
+## Ensures the correct jar file is obtained regardless of orthoinference project version
+orthoinference_jar_file=$(ls target/orthoinference-*-jar-with-dependencies.jar)
```

Comment:
As a thought, should this be the way Jenkinsfiles are organized? Right now jar versions are taken from Global config variables that need to be manually set. Maybe we should just use this wildcard way of doing things so that its less sticky for someone to run steps

Reply:
If it can be done for the Jenkins files, it might be a good idea.  Using the wildcard frees us up from having to manually update the version.

The only thing is we need to be sure only one jar version exists at a time.  Adding it in the shell script is safe since the jar file name is found immediately after doing a `mvn clean` and rebuild ensuring an old `target` directory is removed and regenerated containing only the new jar version.

Reply:
> The only thing is we need to be sure only one jar version exists at a time.

A very good reason to _not_ do this. Just use a variable in the script to store the version number. Or if you really want, have maven update the script (there's probably a way to do this).

Reply:
That's fair.  In the small shell script it's much easier to be sure the jar generated in the previous command is the only one.

In a larger system like Jenkins, it's doable but it might be easier to accidentally have additional jar files.

Reply:
> If it can be done for the Jenkins files, it might be a good idea. Using the wildcard frees us up from having to manually update the version.
> 
> The only thing is we need to be sure only one jar version exists at a time. Adding it in the shell script is safe since the jar file name is found immediately after doing a `mvn clean` and rebuild ensuring an old `target` directory is removed and regenerated containing only the new jar version.

This does also match how the Jenkinsfile does things though, each time a step is run it rebuilds the target folder